integration/spark2/src/test/scala/org/apache/spark/carbondata/TestStreamingTableOperation.scala

```diff
@@ -182,6 +182,10 @@ class TestStreamingTableOperation extends QueryTest with BeforeAndAfterAll {
 
   // normal table not support streaming ingest
   test("normal table not support streaming ingest") {
```

Comment:
Please add alter table description for this test case

Reply:
fixed